jbpm-test-coverage/src/test/java/org/jbpm/test/functional/log/ProcessInstanceLogCleanTest.java

```diff
@@ -223,7 +224,7 @@ public void deleteLogsWithStatusActive() {
 
     @Test
     @BZ("1188702")
-    public void deleteLogsByDate() {
+    public void deleteLogsByDate() throws InterruptedException {
```

Comment:
If only this test is problematic, I would add a delay just here, otherwise we will be prolonging all tests in the class.

Reply:
problem is only happening for that particular test method, however, the problem itself is the call to the method 

`startProcess(kieSession, HELLO_WORLD_PROCESS_ID, 4);`

That's the one which is starting 4 processes at same time - for the code above.

So far, the issue is only happening for that test method, but it might affect future test cases calling the same "startProcess" method.

Another option, would be to create a new method overloaded called "startProcess", passing a delay in miliseconds for instance.

I'd prefer this option - what do you think @MarianMacik?

Reply:
yes, having a different method seems more clear.

One second delay is too much, perhaps 5-10 ms is enough as precision is per millisecond.

Reply:
+1 @afalhambra

Reply:
Done. Updated.